enterprise/server/remote_execution/runner/runner.go

```diff
@@ -486,6 +501,14 @@ func (p *Pool) PullDefaultImage() {
 // The returned runner is considered "active" and will be killed if the
 // executor is shut down.
 func (p *Pool) Get(ctx context.Context, task *repb.ExecutionTask) (*CommandRunner, error) {
+	r, err := p.get(ctx, task)
```

Comment:
just return p.get(ctx, task)?

Reply:
Good catch -- this was an artifact of this func having more stuff in it.